app/components/affiliate-link-page/accept-referral-button.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="flex flex-col items-center justify-center" ...attributes>
+  <PrimaryButtonWithSpinner
+    @size="regular"
+    @shouldShowSpinner={{this.isCreatingAffiliateReferral}}
+    @isDisabled={{not this.acceptOfferButtonIsEnabled}}
+    class="px-5 py-3"
+    {{on "click" this.handleAcceptOfferButtonClick}}
+  >
+    {{#if this.currentUserIsAnonymous}}
+      {{svg-jar "github" class="fill-current w-5 transform transition-all mr-2"}}
+    {{/if}}
+
+    Accept offer →
```

Comment:
Add an empty line before new blocks - better for readability and more consistent with our existing code